### ISSUE TITLE
Add component for matrix

### ISSUE THREAD

**Issue by ank1traj** [WebSorcery/testcase-generator#5]
Generate matrices of various sizes and values based on their input parameters. The component takes input parameters such as the number of rows and columns of the matrix, the minimum and maximum value of the elements in the matrix, and whether the matrix is sparse or dense.

**Comment by aindree-2005** [comment]
@ank1traj can I work on this?

**Comment by aindree-2005** [comment]
@ank1traj unassigned myself because I am unable to run the project on VS Code and I won't be able to finish it by 28th. Sorry!

**Comment by ank1traj** [comment]
@aindree-2005 Okay, would you please tell me why you're not able to run the project on VS Code?

**Comment by aindree-2005** [comment]
> @aindree-2005 Okay, would you please tell me why you're not able to run the project on VS Code?

No idea. Followed the steps from readme. But it keeps navigating to the last project I worked on somehow

**Comment by ank1traj** [comment]
@aindree-2005 Problem is from your end not from the project's side. If it's navigating to your last project then how the problem is from the project?

**Comment by aindree-2005** [comment]
> @aindree-2005 Problem is from your end not from the project's side. If it's navigating to your last project then how the problem is from the project?

Yeah that's what I think.. Maybe a problem with VS Code. That's why I unassigned myself
